packages/vitest/src/integrations/vi.ts

```diff
@@ -370,7 +370,7 @@ function createVitest(): VitestUtils {
 
   const utils: VitestUtils = {
     useFakeTimers(config?: FakeTimerInstallOpts) {
-      if (workerState.isChildProcess) {
+      if (process.send) {
```

Comment:
Maybe create an utility method `isChildProcess()` which describes this check better.

Reply:
I think this check is enough. Or do you want to expose it for public use? Or just to make this assertion clearer?

Reply:
Just to make the check more clear. It's a bit magical to check for `process.send` in code blocks where we don't even use `process.send`. Some may not know that this is how to identify child processes. Makes code more readable.